### DISCUSSION TITLE
Issue #1335 maybe caused by `nnn_cd()`

### DISCUSSION THREAD

**Discussion by nemonein** [jarun/nnn#1882] [Q&A] [answered]
I have the same situation as @FerrisWasTaken on the Issue #1335. After coming from the shell(!), the plugins are not executed at all.

I think the culprit is the function `nnn_cd()`. When I turned it off, plugins work well after returning from the shell.

**Comment by N-R-K** [comment]
I don't understand how that can interfere with plugin launching.

**Comment by nemonein** [comment, marked as answer]
I placed `nnn_cd()` in ~/.oh-my-zsh/custom directory. That is the culprit.
When `nnn_cd` is in `.zshrc`, problem never occurs.
